pytorch_lightning/core/lightning.py

```diff
@@ -1254,7 +1252,7 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx,
         if not isinstance(optimizer, LightningOptimizer):
             # wraps into LightingOptimizer only for running step
             optimizer = LightningOptimizer.to_lightning_optimizer(optimizer, self.trainer)
-        optimizer.step(closure=optimizer_closure, *args, **kwargs)
+        optimizer.step(closure=optimizer_closure)
```

Comment:
Could you explain how the user can provide args/kwargs to his optimizer now? I assume directly to the lightning optimizer?

Reply:
cc: @awaelchli @justusschock